.github/workflows/pr-matrix.yaml

```diff
@@ -0,0 +1,44 @@
+name: Execution Testing
+
+permissions:
+  contents: read
+  checks: write
+
+on:
+  pull_request:
+    branches:
+      - main
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number }}
+  cancel-in-progress: true
+
+jobs:
+  Exec-Tests-Windows:
+    strategy:
+      fail-fast: false
+      matrix:
+        SKU: [windows-intel]
+        TestTarget: [check-hlsl-d3d12, check-hlsl-warp-d3d12, check-hlsl-vk, check-hlsl-clang-d3d12, check-hlsl-clang-warp-d3d12, check-hlsl-clang-vk]
+
+    uses: ./.github/workflows/test-all.yaml
```

Comment:
nit: This file could probably use a rename since it now seems to be a dependency of all the other files